lib/dialects/postgres/query-generator.js

```diff
@@ -417,8 +417,13 @@ var QueryGenerator = {
 
   addLimitAndOffset: function(options) {
     var fragment = '';
-    if (options.limit) fragment += ' LIMIT ' + this.escape(options.limit);
-    if (options.offset) fragment += ' OFFSET ' + this.escape(options.offset);
+
+    if (typeof options.limit !== "undefined" && options.limit !== null) {
```

Comment:
What is the difference between ```(options.limit)``` and ```typeof options.limit !== "undefined" && options.limit !== null```, I think both does same thing?

May be we can use ```(!!options.limit)``` that will be better than checking for all types